### ISSUE TITLE
[1.20.2 v5.0] Crash: Class ... does not have member field 'boolean clientPaused'

### ISSUE THREAD

**Issue by HatTrkPatrk** [NerjalNosk/bettermaps#2]
```
[16:53:33] [main/ERROR]: Minecraft has crashed!
net.fabricmc.loader.impl.FormattedException: java.lang.NoSuchFieldError: Class com.nerjal.bettermaps.Bettermaps does not have member field 'boolean clientPaused'
	at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:63) ~[fabric-loader-0.15.3.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:472) ~[fabric-loader-0.15.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) ~[fabric-loader-0.15.3.jar:?]
	at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:69) ~[fabric-loader-0.15.3.jar:?]
Caused by: java.lang.NoSuchFieldError: Class com.nerjal.bettermaps.Bettermaps does not have member field 'boolean clientPaused'
	at com.nerjal.bettermaps.Bettermaps.<clinit>(Bettermaps.java:49) ~[bettermaps-mc1.20-0.5.jar:?]
	at net.minecraft.world.GameRules.handler$zeg000$bettermaps$staticInitInjector(GameRules.java:526) ~[server-intermediary.jar:?]
	at net.minecraft.world.GameRules.<clinit>(GameRules.java:126) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.<clinit>(MinecraftServer.java:182) ~[server-intermediary.jar:?]
	at net.minecraft.server.SaveLoading$DataPacks.load(SaveLoading.java:97) ~[server-intermediary.jar:?]
	at net.minecraft.server.SaveLoading.load(SaveLoading.java:29) ~[server-intermediary.jar:?]
	at net.minecraft.server.Main.method_43612(Main.java:159) ~[server-intermediary.jar:?]
	at net.minecraft.util.Util.waitAndApply(Util.java:945) ~[server-intermediary.jar:?]
	at net.minecraft.util.Util.waitAndApply(Util.java:939) ~[server-intermediary.jar:?]
	at net.minecraft.server.Main.main(Main.java:157) ~[server-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.3.jar:?]
	... 3 more
```

Running version 5.0, obtained from Modrinth.

Full latest.log: https://gist.github.com/hattrickpatrick/ccb30232acca2b24d5b8853be94fb4be

**Comment by NerjalNosk** [comment]
Could you please try again using the snapshot build I just rendered available in a [new pre-release](https://github.com/NerjalNosk/bettermaps/releases/tag/0.5.1-SNAPSHOT-1.20), linked below?
I have a feeling this might be due to a class property not being found by the server as it was looking for it, while it was marked as client-side (although it shouldn't even be looking for it, but that I do not know why)

https://github.com/NerjalNosk/bettermaps/releases/download/0.5.1-SNAPSHOT-1.20/bettermaps-mc1.20-0.5.1-SNAPSHOT.jar

**Comment by NerjalNosk** [comment]
Fixed in 0.5.1
